package.json

```diff
@@ -1,10 +1,40 @@
 {
-    "private": true,
-    "name": "o-brand",
-    "devDependencies": {
-        "eslint": "^7.9.0",
-        "eslint-config-origami-component": "^2.1.0",
-        "origami-ci-tools": "^2.1.0",
-        "stylelint-config-origami-component": "^1.0.4"
+  "name": "@financial-times/o-brand",
+  "version": "100.0.0-13",
+  "description": "Tools to \"brand\" other Origami components.",
+  "keywords": [],
+  "homepage": "https://github.com/Financial-Times/o-brand#readme",
+  "bugs": {
+    "url": "https://github.com/Financial-Times/o-brand/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Financial-Times/o-brand.git"
+  },
+  "license": "MIT",
+  "author": "",
+  "type": "module",
+  "exports": {
+    ".": {
+      "browser": "./main.js"
     }
+  },
+  "browser": "./main.js",
+  "types": "./types/main.d.ts",
```

Comment:
This branch has more changes than the minimum required for origami v2 components such as this "types" property and the "exports" property